xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp

```diff
@@ -579,6 +579,19 @@ bool CDecoder::Open(AVCodecContext* avctx, AVCodecContext* mainctx, const enum A
       mainctx->slice_flags = SLICE_FLAG_CODED_ORDER|SLICE_FLAG_ALLOW_FIELD;
       mainctx->hwaccel_context = &m_hwContext;
 
+      // update processInfo
+      std::list<EINTERLACEMETHOD> deintMethods;
```

Comment:
wrong place. like in vaapi, this needs to be done in COutput. there's different modes available depending on whether mixer is used or not.